### ISSUE TITLE
PNG support

### ISSUE THREAD

**Issue by zeitt** [Team-Innis/UtH-Engine#4]

**Comment by DrJonki** [comment]
Implemented in 85dfab3800736656a9fd35bb8890d044b5dd6e48. Needs testing.

**Comment by DrJonki** [comment]
File format support (from stb_image.h):

```
JPEG baseline (no JPEG progressive)
PNG 8-bit only

TGA (not sure what subset, if a subset)
BMP non-1bpp, non-RLE
PSD (composited view only, no extra channels)

GIF (*comp always reports as 4-channel)
HDR (radiance rgbE format)
PIC (Softimage PIC)
```
